apps/hashdotdev/package.json

```diff
@@ -47,22 +49,23 @@
     "remark-parse": "10.0.1",
     "sharp": "0.31.3",
     "slugify": "1.6.5",
+    "typescript": "5.1.6",
```

Comment:
Agree it should remain in dev dependencies, but the reason I added it here is because in all other TS packages we are putting it into the dependencies, and making it consistent allowed me to use the existing `fix:dependency-version-consistency` script to programmatically update all other `package.json` files.

I could also move it into the dev dependencies for all packages though?